virttest/utils_kernel_module.py

```diff
@@ -0,0 +1,117 @@
+import os
```

Comment:
There is a missing copyright/author/license (yes, it's missing on many other places, but we should be adding them at least to the new modules. You can inspire from `avocado_vt` folder or at least the `:copyright:` tag that is used in `qemu_vm.py`)